thrift/src/main/resources/META-INF/plugin.xml

```diff
@@ -105,7 +105,7 @@
       ]]>
   </change-notes>
   <vendor>@fedor</vendor>
-  <idea-version since-build="192"/><!-- Will be patched by the plugin -->
+  <idea-version since-build="201"/><!-- Will be patched by the plugin -->
```

Comment:
Please revert since `thrift:patchPluginXml` task automatically updates it. I added the comment to reflect it:

> Will be patched by the plugin